### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by Bhuvaneshwari29**
Completed JS1 week3 homework.

### CODE REVIEW

File: javascript/javascript1/week3/js1week3.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+//get a note
+function getNote(id) {
+  for(i=0;i<notes.length;i++)
+  {
+    if(id===isNaN()){
```
Comment (florinmaticu) [inline javascript/javascript1/week3/js1week3.js:99]: you could check if an id is a number before you start the loop

File: javascript/javascript1/week3/js1week3.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      return 'you entered valid format';
+    }
+    else if (id===notes[i].id){
+      return notes[i];
+    }
+    else{
+      return error;
```
Comment (florinmaticu) [inline javascript/javascript1/week3/js1week3.js:106]: you did not defined the `error` and you are using it here.

File: javascript/javascript1/week3/js1week3.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+//CactusIO-interactive (Smart phone usage app) optional
+//adding an activity
+let activities=[];
+function addActivity(dt,act,dur){
+  let addedActivities = { date:new Date(),
+    activity:act,
+  duration:dur
+};
+if(typeof act==='string' && typeof dt==='string' && typeof dur==='number'){
```
Comment (florinmaticu) [inline javascript/javascript1/week3/js1week3.js:134]: you could do the check at the beginning of the function

File: javascript/javascript1/week3/js1week3.js [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+logOutNotesFormatted(); // should log out the text below
+// The note with id: 1, has the following note text: Pick up groceries
+// The note with id: 2, has the following note text: Do laundry
+
+//CactusIO-interactive (Smart phone usage app) optional
+//adding an activity
+let activities=[];
+function addActivity(dt,act,dur){
+  let addedActivities = { date:new Date(),
```
Comment (florinmaticu) [inline javascript/javascript1/week3/js1week3.js:130]: i assume you want to do add here `dt` ?
```suggestion
  let addedActivities = { date:dt,
```

File: javascript/javascript1/week3/js1week3.js (hunk 1 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript1/week3/js1week3.js]: Yes..Done

File: javascript/javascript1/week3/js1week3.js (hunk 2 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript1/week3/js1week3.js]: Defined the error

File: javascript/javascript1/week3/js1week3.js (hunk 3 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript1/week3/js1week3.js]: Yes ...updated

File: javascript/javascript1/week3/js1week3.js (hunk 4 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript1/week3/js1week3.js]: Yes..You are right. Changed !!

**Review by florinmaticu — APPROVED** [review]
Good job

**Pull request merged**